messages/colorgradient/colorgradient.bg-BG.yml

```diff
@@ -1,25 +1,31 @@
 kendo:
   colorgradient:
+    # The aria-label applied to the ColorGradient component when the value is empty
+    colorGradientNoColor: Colorgradient no color chosen
+
     # The title for the gradient color drag handle chooser
-    colorGradientHandle: Избери цвят
+    colorGradientHandle: Choose color
+
+    # The title for the clear button
+    clearButton: Clear value
 
     # The title for the hue slider handle
-    hueSliderHandle: Добави оттенък
+    hueSliderHandle: Set hue
```

Comment:
not needed, could leave the BG translation. Also applicable to the below BG translations.